codechecker_common/output/json.py

```diff
@@ -0,0 +1,24 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+"""Helper and converter functions for json output format."""
+
+from typing import Dict
+from codechecker_common.report import Report
+
+
+def to_parse_output_json(report: Report) -> Dict:
```

Comment:
In case of codeclimate this function is called as `convert_reports` , in case of gerrit it is called as `create_json` and in case of json it is called as `to_parse_output_json`. Wouldn't it be better to use a similar function name for each format?

Something like: `to_<format>` or `convert_to_<format>` (e.g.: `to_codeclimate`, `to_gerrit`, `to_json`)?